src/server.js

```diff
@@ -0,0 +1,98 @@
+var fs = require('fs'),
+  dns = require('native-dns'),
+  dnsServer = dns.createServer(),
+  https = require('https'),
+  records = {
+    TXT: {},
+    A: {}
+  },
+  types = {
+    1: 'A',
+    16: 'TXT'
+  },
+  apiServer;
+
+exports.serve = function(certDir, dnsPort, apiPort) {
+  dnsServer.on('request', function (request, response) {
+    var type = types[request.question[0].type];
+    var host = request.question[0].name.toLowerCase();
+    if (records[type] &&
+        records[type][host]) {
+      response.answer.push(dns[type]({
+        name: host,
+        address: records[type][host], //for A records
+        data: [records[type][host]], //for TXT records
+        ttl: 600,
+      }));
+    }
+    response.send();
+  });
+
+  dnsServer.on('error', function (err, buff, req, res) {
```

Comment:
`buff`, `req` and `res` are not used.

Do we want to inform the client about the error?